inlang/source-code/sdk2/src/project/util/insertBundleNested.ts

```diff
@@ -0,0 +1,47 @@
+import type { BundleNested } from "../../schema/schemaV2.js";
+import type { InlangProject } from "../api.js";
+import { json } from "./toJSONRawBuilder.ts";
+
+export const insertNestedBundle = async (
+	project: InlangProject | undefined,
+	bundle: BundleNested
+): Promise<void> => {
+	if (project === undefined) {
+		throw new Error("Project is undefined");
+	}
+	await project.db
+		.insertInto("bundle")
+		.values({
+			id: bundle.id,
+			alias: json(bundle.alias),
+		})
+		.returning("id")
+		.execute();
```

Comment:
Suggested changes to make the function generalizable/available for inlang apps:

1. I would move the function to `query-utilities`
2. I would not let the function .execute() and define what should be returned (leave it to apps)
3. I would use `db` as the argument instead of `project` (you don't need project)
4. Why can `project`/`db` be undefined?

Reply:
1. greate
2. we have multiple executes in the function, one per bundle/message/variant... we could wrap them into a transaction or what would you return?
3. just copied from fink2 - which was copied from the original prototype :D - guess thats a leftover - fixed in next commit
4. see 3, done